certs/ca.crt

```diff
@@ -0,0 +1,28 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
We need to handle these differently later on. I mean the certs on git is not right

Reply:
Yeah please let me know how you would like it to be handled. There is a readme guide how to generate certs, so everybody can generate their own certs if they want.

Reply:
I think for now, this is fine, we can pick this up later on! Just wanted to flag it up so that we dont forget